api/src/generate-service-worker.js

```diff
@@ -56,60 +56,64 @@ const getLoginPageContents = async () => {
   } catch (err) {
     logger.error('Error rendering login page %o', err);
     // default to returning the file
-    return [path.join(apiSrcDirectoryPath, 'templates', 'login', 'index.html')];
+    return [path.join(apiSrcDirectoryPath, 'templates', 'login', 'index.html')]; // TODO figure out if this is right
```

Comment:
This will be tested on the server, right? Just making sure the `// TODO` wasn't forgotten :)

Reply:
I checked and this code was unreachable (the other code never throws).